lblod/app-digitaal-loket
# [ Toezicht Module ] Bundling new worship forms

# Description
DL-5185 & DL-5186 & DL-5187 & DL-3748

This PR adds new worship forms and their decidableBy rules : 

- Reactie op opvragen bijkomende inlichtingen door de toezichthouder (gemeente/provincie) aan de eredienstbesturen (Decidable by EB or CB)

- Aanvraag desaffectatie presbyteria/kerken (Decidable by Gemeente)

- Opvragen bijkomende inlichtingen eredienstbesturen (Decidable by Gemeente or Provincie)

It also enables "Desaffectatie presbyterium" form in Kalliope and forbids the export for "Reactie op opvragen bijkomende inlichtingen door de toezichthouder (gemeente/provincie) aan de eredienstbesturen" & "Opvragen bijkomende inlichtingen eredienstbesturen"

It also enables the rule for Kalliope & bundles the following form 

- Gezamenlijk-voorstel-tot-fusie


# Type of change


# Related Apps

- app-digitaal-loket
- app-worship-decisions-database
- app-public-decisions-database
- app-toezicht-abb
- app-meldingsplichtige-api

# Related Services

- prepare-submissions-for-export-service
- enrich-submission-service
- worship-submissions-graph-dispatcher-service

# How to test 

1. Log as Gemeente (for form "Aanvraag desaffectatie presbyteria/kerken" and "Opvragen bijkomende inlichtingen eredienstbesturen") as EB or CB for "Reactie op opvragen bijkomende inlichtingen door de toezichthouder (gemeente/provincie) aan de eredienstbesturen"
2. Create new submissions in toezicht module and send them
3. Log `prepare-submissions-for-export-service` to check if they're exported they should also be visible in `delta-producer-publication-graph-maintainer-worship-submissions`
4. Connect loket and consumer stacks (Toezicht-abb / worship-decisions-database) See tutorial in Wiki
5. They should be visible in consumer apps based on rules

# Links to other PR's

- https://github.com/lblod/manage-submission-form-tooling/pull/38
- https://github.com/lblod/app-worship-decisions-database/pull/47
- https://github.com/lblod/app-toezicht-abb/pull/32
- https://github.com/lblod/app-meldingsplichtige-api/pull/34
- https://github.com/lblod/app-public-decisions-database/pull/19
- https://github.com/lblod/prepare-submissions-for-export-service/pull/10
- https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/13


# Notes

Bundle `prepare-submissions-for-export-service` bump

- drc restart migrations berichtencentrum-sync-with-kalliope
